mix.exs

```diff
@@ -45,7 +45,7 @@ defmodule AbsintheRelay.Mixfile do
 
   defp deps do
     [
-      {:absinthe, "~> 1.5.0"},
+      {:absinthe, "~> 1.5"},
```

Comment:
I would prefer to do `{:absinthe, "~> 1.5.0 or ~> 1.6.0"}`. This package tends to rely on absinthe schema building internals which can change between minor releases.

Reply:
Well, is it OK now?

Reply:
No, we want to support exactly `"~> 1.5.0 or ~> 1.6.0"` because the current version will work on both, upgrading to 1.6.0 is not mandatory.

Reply:
OK, done.